All EIPIP Meetings/Meeting 065.md

```diff
@@ -0,0 +1,472 @@
+
+**Sam**
+* Some editors want to force all assets to be cc0. And I, I don't want to do that. I want anything that isn't, copy left to be allowed for assets. 
+
+**Pooja Ranjan**
+* Greg, do you have any thoughts on this? 
+
+**Golvin**
+* I'm not a lawyer. 
+
+**Pooja Ranjan**
+* Again,. All right. I mean, on a high level, I personally think that when we are talking Ethereum and we are talking about having something as a standard and giving place Ethereum repository, we should try to make, things, as much as open source, free, no patent, no, like trademark things, so people can use it freely, Ethereum is an open source blockchain. We are like trying to, I mean, anyone is welcome to come and use it, so why to add things which may or may not have, permissions to be used or reused, actually. But that's my thought on that. I would be happy to hear other, editors thought on that. 
+
+**Sam**
+* Yeah, that, that, that's my thought as well. Like if somebody wants to come in with a, and and mandate that you have to use a certain algorithm in an EIPs and then, you know, a year later they start suing everybody, because they patented that algorithm, I think that's completely ridiculous and we should try to avoid that situation. 
+
+**Pooja Ranjan**
+* Right. And, now we are joined by Pandapip, Panda. We were discussing a couple of proposals, by you, issues that you have listed, and one of the latest one, which is currently in a discussion is about the, patent and also the use of license for EIPs assets. I wonder if you have any thoughts to share on that? 
+
+**Pandapip**
+* Sorry, didn't catch that second part. 
+
+**Pooja Ranjan**
+* So it's about the, license usage for EIPss
+
+**Pandapip**
+* I'm right next to a road right now when I'm outside and on really, really, really bad internet and a bad computer. 
+
+**Pooja Ranjan**
+* Okay, Not a problem. if you can hear me now, we were discussing about, if we should be allowing patenting, staff or maybe assets, which may have some trademark already or should it be only, which is free for people to reuse license issues. If you have any thoughts on that, probably you can share. 
+
+**Pandapip**
+* Well, I think it's pretty obvious that, well, I mean, for one thing, all the, all of the, assets should be, sorry, all the EIPs should be cc0, just so that anyone can just use an EIPs  without having to worry about randomly being sued, like in any case. and I feel like patent and trademark, waivers are just like kind of an extension of that. You should be able to use an EIPs without having to worry about randomly being sued for using it. 
```

Comment:
```suggestion
* Well, I think it's pretty obvious that, well, I mean, for one thing, all of the, assets should be, sorry, all the EIPs should be cc0, just so that anyone can just use an EIPs  without having to worry about randomly being sued, like in any case. and I feel like patent and trademark, waivers are just like kind of an extension of that. You should be able to use an EIPs without having to worry about randomly being sued for using it. 
```